docs/designs/credential.md

```diff
@@ -42,14 +43,31 @@ Authorization: Bearer {token}
 The value of secrets object can be an object to provide more details of the request header. It can be configured in `docfx.yml` like:
 ```yml
 secrets:
-  "https://contoso.com/path/file":
-    token: "?sig={token}"
+  http:
+  - location: "https://contoso.com/path/file":
```

Comment:
is this _base URL_ or _exact URL_, using location feels like it is an exact URL, but we want to just qualify base URL, right?

Reply:
Changed to `baseUrl`. We can also use it for *exact URL*